template/apps/web/package.json

```diff
@@ -63,12 +63,17 @@
     "prettier-config-custom": "workspace:*",
     "storybook": "8.2.0",
     "storybook-dark-mode": "4.0.2",
+    "stylelint": "16.7.0",
+    "stylelint-config-standard-scss": "13.1.0",
     "tsconfig": "workspace:*",
     "typescript": "5.2.2"
   },
   "lint-staged": {
     "*.{ts,tsx}": [
       "eslint --fix"
+    ],
+    "*.{css,scss}": [
+      "stylelint"
```

Comment:
```suggestion
      "stylelint --fix"
```

<img width="862" alt="Screenshot 2024-07-16 at 2 57 37 PM" src="https://github.com/user-attachments/assets/45dcc867-430e-4f98-ba04-65773ce0fe1f">

from the docs of lint staged https://github.com/lint-staged/lint-staged
I think we can apply autofix, if we use it in eslint